frontend/components/Navigation/Navigation.tsx

```diff
@@ -55,7 +61,25 @@ interface Props {
   activeFolder?: string;
 }
 
-const Navigation = ({ workspace, folders, activeFolder }: Props) => {
+const Navigation: FC<Props> = ({ workspace, folders, activeFolder }) => {
+  const createFolder = {
+    id: "create-folder",
+    title: "Create new folder",
+    description: "create folder",
+    workspace: workspace.id,
+  };
+
+  const icons: { [key: string]: string } = {
+    closed: require("../../public/folderClosed.svg"),
+    open: require("../../public/folderOpen.svg"),
+  };
+
+  const alphabetisedFolders = folders.sort((a, b) => {
+    const textA = a.title.toUpperCase();
+    const textB = b.title.toUpperCase();
+    return textA < textB ? -1 : textA > textB ? 1 : 0;
```

Comment:
These are strings, so you could use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) instead:
```suggestion
    return textA.localeCompare(textB);
```

Also I just learned that this function supports case-insensitive comparisons. So you can replace this entire sorting logic with:

```
folders.sort((a, b) => a.localeCompare(b, 'en', { sensitivity: 'base' }))
```

Reply:
Well, I just realized I forgot the `title` property. It should be:
```ts
folders.sort((a, b) => a.title.localeCompare(b.title, 'en', { sensitivity: 'base' }))
```